SharedPackages/BrowserServicesKit/Tests/CommonTests/Extensions/URLExtensionTests.swift

```diff
@@ -68,8 +68,8 @@ final class URLExtensionTests {
         }
 
         let url = rawValue.decodedURL
-        #expect(url != nil, sourceLocation: .init(fileID: #fileID, filePath: #filePath, line: Int(line), column: 0))
-        #expect(url!.isValid, sourceLocation: .init(fileID: #fileID, filePath: #filePath, line: Int(line), column: 0))
+        #expect(url != nil, sourceLocation: .init(fileID: #fileID, filePath: #filePath, line: Int(line), column: 1))
+        #expect(url!.isValid, sourceLocation: .init(fileID: #fileID, filePath: #filePath, line: Int(line), column: 1))
```

Comment:
This was a change within the Swift Testing library where a line or column of `0` causes a fatal error.